docs/source/release/index.rst

```diff
@@ -11,7 +11,7 @@ Release Notes
 
    These release notes are for versions of ibis **1.0 and later**. Release
    notes for pre-1.0 versions of ibis can be found at :doc:`release-pre-1.0`
-
+* :bug:`2342` Update impala kerberos dependencies
```

Comment:
You need to leave a blank line before this (otherwise rst doesn't understand this is the beginning of a list).

Reply:
Got it, thanks for the heads up!